pylintrc

```diff
@@ -55,7 +55,7 @@ extension-pkg-whitelist=alsaaudio,pyA20
 # E1103: %s %r has no %r member (but some types could not be inferred) - fails to infer real members of types, e.g. in Celery
 # W0231: method from base class is not called - complains about not invoking empty __init__s in parents, which is annoying
 # R0921: abstract class not referenced, when in fact referenced from another egg
-disable=mixed-indentation,line-too-long,bad-continuation,missing-docstring,too-many-nested-blocks,too-many-branches,no-self-use,global-statement,too-many-statements
+disable=mixed-indentation,line-too-long,bad-continuation,missing-docstring,too-many-nested-blocks,too-many-branches,no-self-use,global-statement,too-many-statements,duplicate-code,relative-import
```

Comment:
If I checked correctly, it's only a few places in main.py - the command line arguments and one line in orangepi.py. We can ignore tunein.py. It's an extenal thing that we need to pull new version of and stuff anyway.